src/player.js

```diff
@@ -813,20 +813,7 @@ export default class Player extends FakeEventTarget {
    * @public
    */
   set currentTime(to: number): void {
-    if (this._engine) {
-      if (Utils.Number.isNumber(to)) {
-        let boundedTo = to;
-        if (to < 0) {
-          boundedTo = 0;
-        }
-        const safeDuration = this.isLive() ? this._engine.duration : this._engine.duration - DURATION_OFFSET;
-
-        if (boundedTo > safeDuration) {
-          boundedTo = safeDuration;
-        }
-        this._engine.currentTime = boundedTo;
-      }
-    }
+    this._playbackMiddleware.setCurrentTime(to, this._setCurrentTime.bind(this, to));
```

Comment:
do you need to pass to to the bind as well, if it's already passed as a parameter to setCurrentTime ?